### PR TITLE
Polish

### PR DESCRIPTION
**Pull request by izeye**
This PR fixes some typos and polishes trivial stuff.

### CODE REVIEW

File: buildSrc/src/test/java/org/springframework/boot/build/context/properties/ConfigurationPropertiesTests.java [hunk 1]
```diff
@@ -32,7 +32,7 @@
 class ConfigurationPropertiesTests {
 
 	@Test
-	void whenJsonHasAnIntegerDefaultValueThenItRemainsAnIntegerWhenRead() {
+	void whenJsonHasAnIntegerDefaultValueThenItRemainsAsIntegerWhenRead() {
```
Comment (wilkinsona) [inline buildSrc/src/test/java/org/springframework/boot/build/context/properties/ConfigurationPropertiesTests.java:35]: `An` is correct here, IMO.

**Review by wilkinsona — CHANGES_REQUESTED** [review]
Thanks very much, @izeye. There's one change that I don't agree with. WDYT?

File: buildSrc/src/test/java/org/springframework/boot/build/context/properties/ConfigurationPropertiesTests.java (hunk 1 above)
Comment (izeye) [reply buildSrc/src/test/java/org/springframework/boot/build/context/properties/ConfigurationPropertiesTests.java]: @wilkinsona Thanks for the quick feedback!

I didn't know that a noun could come after "remain". After searching it a bit, I found I was wrong. I reverted it in 2420f0c.

**Comment by snicoll** [comment]
Thanks for the follow-up @izeye!

**Pull request closed without merging**